ngmix/tests/test_metacal.py

```diff
@@ -42,6 +42,40 @@ def test_metacal_smoke(psf):
         assert np.all(obs_dict['noshear'].image != obs_dict['2m_psf'].image)
 
 
+@pytest.mark.parametrize('psf', ['gauss', 'fitgauss', 'galsim_obj'])
+@pytest.mark.parametrize('send_rng', [True, False])
+def test_metacal_send_rng(psf, send_rng):
+
+    rng = np.random.RandomState(seed=100)
+    obs = _get_obs(rng, noise=0.005)
+
+    if psf == 'galsim_obj':
+        psf = galsim.Gaussian(fwhm=0.9)
+
+    kw = {
+        'psf': psf, 'fixnoise': False, 'types': ['noshear', '1p', '1m'],
+    }
+    if send_rng:
+        kw['rng'] = rng
+
+    if psf == 'fitgauss' and not send_rng:
+        with pytest.raises(ValueError):
+            obs_dict1 = ngmix.metacal.get_all_metacal(obs, **kw)
```

Comment:
I thought we never needed the RNG. Is this not true?

Reply:
It is needed for fitgauss to generate guesses for the fitter